daemon/networkdriver/bridge/driver.go

```diff
@@ -140,6 +194,16 @@ func InitDriver(job *engine.Job) engine.Status {
 		if err := ioutil.WriteFile("/proc/sys/net/ipv4/ip_forward", []byte{'1', '\n'}, 0644); err != nil {
 			job.Logf("WARNING: unable to enable IPv4 forwarding: %s\n", err)
 		}
+
+		if enableIPv6 {
+			// Enable IPv6 forwarding
+			if err := ioutil.WriteFile("/proc/sys/net/ipv6/conf/default/forwarding", []byte{'1', '\n'}, 0644); err != nil {
+				job.Logf("WARNING: unable to enable IPv6 default forwarding: %s\n", err)
+			}
+			if err := ioutil.WriteFile("/proc/sys/net/ipv6/conf/all/forwarding", []byte{'1', '\n'}, 0644); err != nil {
```

Comment:
It's not the same.  IPv4 doesn't have router advertisements -- it usually uses DHCP.  DHCPv6 is a lot less common, because it requires more setup.  Router advertisements require zero configuration on any of the nodes -- it "just works", so it's the more common network setup solution.